grouper/api/handlers.py

```diff
@@ -176,6 +179,33 @@ def get(self, *args, **kwargs):
             )
 
 
+class UserMetadata(GraphHandler, ListUsersUI):
+    def listed_users(self, users):
+        # type: (Dict[str, User]) -> None
+        users_dict = {}  # type: Dict[str, Dict[str, Any]]
+        for user, data in iteritems(users):
+            metadata = [{"key": m.key, "value": m.value} for m in data.metadata]
```

Comment:
Yeah, good call, since I'm changing the JSON schema anyway, I may as well change this. We have an index that requires there only be one setting per metadata key, so nothing lost there.